src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,38 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List<Machine> get() {
+        Bulldozer bulldozer1 = new Bulldozer();
+
+        bulldozer1.setName("Bulldozer 1");
+        bulldozer1.setColor("Black");
+        bulldozer1.setWeight(300);
+        bulldozer1.setDumpDeepening(1);
+        bulldozer1.setHeight(3);
```

Comment:
here and below, you can make an overloaded constructor (not removing the default one) and put all values there.